### ISSUE TITLE
HTTPError when requesting INSEE dataflow

### ISSUE THREAD

**Issue by beberking** [khaeru/sdmx#21]
Dear developper,

I get a HTTPError when requesting INSEE dataflow.
```python
import sdmx
insee  = sdmx.Request('INSEE')
insee.dataflow()
```
results in
```python
>>> insee.dataflow()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bbk/.local/lib/python3.8/site-packages/sdmx/api.py", line 414, in get
    response.raise_for_status()
  File "/usr/lib/python3/dist-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error:  for url: https://www.bdm.insee.fr/series/sdmx/dataflow/INSEE/latest
```
However, please note that INSEE does support the dataflow request, at the following addresses:
[https://www.bdm.insee.fr/series/sdmx/dataflow/ALL/latest](https://www.bdm.insee.fr/series/sdmx/dataflow/ALL/latest)
[https://www.bdm.insee.fr/series/sdmx/dataflow/FR1/latest](https://www.bdm.insee.fr/series/sdmx/dataflow/FR1/latest)

I guess a hook similar to ISTAT's would fix it, but not sure how.

Cheers,
Bertrand

**Comment by khaeru** [comment]
@beberking good find and thanks for the report! This will be a simple fix, exactly as you say.

From the information you have given, a work around in the meantime is to use either of the following lines:
```
…
insee.dataflow(provider="ALL")
insee.dataflow(provider="FR1")
```
The first returns a slightly larger set.

**Comment by khaeru** [comment]
Version 1.5.0 will be released after #24 is merged, and will include this fix.
